static/app/views/performance/traceDetails/content.tsx

```diff
@@ -481,7 +481,7 @@ function OnlyOrphanErrorWarnings({orphanErrors}: OnlyOrphanErrorWarningsProps) {
         <BannerTitle>{t('Connect the Dots')}</BannerTitle>
         <BannerDescription>
           {t(
-            "If you haven't already, configure performance monitoring to learn more about how your services are interacting with each other. This will provide more clarity about how your errors are linked."
+            "If you haven't already, configure tracing to learn more about how your services are interacting with each other. This will provide more clarity about how your errors are linked."
```

Comment:
Open for suggestions: Should we write Tracing with a capital letter so it is more clear that this is a product?

Reply:
> Just one observation: we seem to be a bit inconsistent with capitalization - mixing Tracing and tracing throughout the frontend

@shellmayr I was adding some comments myself about this. Should we just capitalize everything?

Reply:
Yes that is a good point but I will leave it up to the performance folks. Also maybe we want to do that in a follow up? I am sure there will be other places where a capitalize 'Tracing" will be better

Reply:
Yeah you're making a great point here @priscilawebdev - there may also be places where different capitalizations are actually useful, like referring to the generic functionality "tracing" vs the product "Tracing"

Reply:
I would leave tracing but make the word a link to our docs